src/sentry/incidents/endpoints/organization_alert_rule_index.py

```diff
@@ -195,6 +202,14 @@ def get(self, request: Request, organization) -> Response:
                 ProjectUptimeSubscriptionMode.AUTO_DETECTED_ACTIVE,
             ),
         )
+        crons_rules = Monitor.objects.filter(
```

Comment:
worth writing a quick comment about the monitor environment sorting business, or putting it in a helper method with a docstring

Reply:
Will add after merging I think